Sources/XCTest/XCTAssert.swift

```diff
@@ -52,25 +52,19 @@ private enum _XCTAssertionResult {
     case UnexpectedFailure(ErrorProtocol)
 
     var expected: Bool {
-        switch (self) {
-        case .UnexpectedFailure(_):
-            return false
-        default:
-            return true
+        switch self {
+        case .UnexpectedFailure(_): return false
```

Comment:
Makes sense to me. Thanks, and sorry I didn't realize you had already explained in the commit messages!